### PR TITLE
Split lvm_snapshots role into separate roles

### PR DESCRIPTION
**Pull request by ygalblum**
Split into three roles - create, revert and remove 
For create allow running only check
Do not use check_for_resize as the functionality was moved to shrink_lv

### CODE REVIEW

**Review by swapdisk — CHANGES_REQUESTED** [review]
Also need to update the Roles section of the main README.

**Comment by ygalblum** [comment]
> Also need to update the Roles section of the main README.

Done

File: roles/create_snapshot/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- name: Check available disk space
+  ansible.builtin.include_tasks: check.yml
+
+- name: Create Snapshot
+  vars:
+    create_snapshot_volumes: "{{ create_snapshot_check_status.stdout | from_json }}"
```
[the remark this answers is not in the document]
Comment (swapdisk) [reply roles/create_snapshot/tasks/main.yml:6]: Right, this failure was because of the ansible issue, not a problem with the role.

**Review by swapdisk — APPROVED** [review]
LGTM.

**Pull request merged**